Unity3D/Assets/RosSharp/Scripts/OdometrySubscriber.cs

```diff
@@ -28,12 +29,14 @@ public class OdometrySubscriber : MonoBehaviour
         public GameObject UrdfModel;
```

Comment:
You need the GameObject UrdfModel below only to find its component OdometryTransformManager.
So please delete it and make OdometryTransformManager public.